addons/website/tests/test_website_visitor.py

```diff
@@ -324,3 +325,113 @@ def test_visitor_archive(self):
         new_visitor = self.env['website.visitor'].search([('id', '=', new_visitor.id)])
         self.assertEqual(len(new_visitor), 0, "The new visitor should be deleted when visitor authenticate once again.")
         self.assertEqual(old_visitor.active, True, "The old visitor should be reactivated when visitor authenticates once again.")
+
+@tagged('-at_install', 'post_install')
+class TestPortalWizardMultiWebsites(HttpCase):
+    def test_portal_wizard_multi_websites(self):
```

Comment:
Great to have a complete test but it would be better to slit it (one for each scenario `N)`) with a setup function IMO

Reply:
Adapted :+1: